boards/raytac/mdbt50q_cx_40_dongle/Kconfig

```diff
@@ -0,0 +1,20 @@
+# Raytac MDBT50Q-CX Dongle NRF52840 board configuration
+#
+# Copyright (c) 2018-2023 Nordic Semiconductor ASA
+# Copyright (c) 2025 Raytac Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_RAYTAC_MDBT50Q_CX_40_DONGLE
+
+config BOARD_HAS_NRF5_BOOTLOADER
+	bool "Board has nRF5 bootloader"
+	default y
+	help
+	  If selected, applications are linked so that they can be loaded by Nordic
+	  nRF5 bootloader.
+
+config BOARD_SERIAL_BACKEND_CDC_ACM
+	bool "USB CDC"
+	default y
```

Comment:
this is based on an old version of zephyr, please use the current nrf52840dongle and create this. Also apply changes from https://github.com/zephyrproject-rtos/zephyr/pull/89017/files